courses/api.py

```diff
@@ -275,6 +275,7 @@ def send_enrollment_emails():
                 # Case (Upgrade): When user was enrolled in free mode and now enrolls in paid mode (e.g. Verified)
                 # So, User has an active enrollment and the only changing thing is going to be enrollment mode
                 if enrollment.active and enrollment_mode_changed:
+                    enrollment.edx_enrolled = edx_request_success
```

Comment:
This line could be moved to line 275  as if and elif blocks now have the same exact line

Reply:
Nice!  Good catch